internal/cli/cmd/cluster/buildx.go

```diff
@@ -56,8 +63,26 @@ func newSetupBuildxCmd(cmdName string) *cobra.Command {
 			return err
 		}
 
+		ctx, cancel := context.WithCancel(ctx)
```

Comment:
See also https://github.com/namespacelabs/foundation/blob/fe39ff7287121c5ef3da97c94055953046684c4b/internal/cli/fncobra/sigint.go#L16-L17 for an example of a composable version of this.

Reply:
Ah! Thanks for the reference. I see that `WithSigIntCancel` is already wired in 
https://github.com/namespacelabs/foundation/blob/cleanup-buildx-proxy/internal/cli/fncobra/do.go#L67
Which eventually is used by 
https://github.com/namespacelabs/foundation/blob/cleanup-buildx-proxy/internal/cli/fncobra/main.go#L62

So we can remove this bit and change the signal from SIGTERM to SIGINT.